### PR TITLE
docs: updated cookbook title.

### PR DESCRIPTION
**Pull request by koolhead17**

### CODE REVIEW

**Comment by harshavardhana** [comment]
ping @deekoder

File: docs/aggregate-apache-logs-with-fluentd-and-minio.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Aggregate Apache Logs with fluentd plugin to Minio [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# Aggregate Apache Logs with fluentd plugin for Minio [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (harshavardhana) [inline docs/aggregate-apache-logs-with-fluentd-and-minio.md]: Fix gitter link.

**Comment by harshavardhana** [comment]
ping @koolhead17 can you fix gitter link?

File: docs/aggregate-apache-logs-with-fluentd-and-minio.md (hunk 1 above)
Comment (koolhead17) [inline docs/aggregate-apache-logs-with-fluentd-and-minio.md]: done.

**Comment by harshavardhana** [comment]
You have to resolve conflicts as well @koolhead17

**Comment by koolhead17** [comment]
This can be closed via https://github.com/minio/cookbook/pull/140

**Comment by NitishT** [comment]
Closing this in favour of #140

**Pull request closed without merging**